09_employment/compute_metrics_wage_ratio_2022.do

```diff
@@ -0,0 +1,168 @@
+****************************/
+
+local raw "C:\Users\KMartinchek\Documents\umf-2024\mobility-from-poverty\09_employment"
+local wages "C:\Users\KMartinchek\Documents\umf-2024\mobility-from-poverty\09_employment"
+
+/***** save living wage as .dta *****/
+cd `wages'
+import delimited using "mit-living-wage-2022.csv", clear // while in code review, please note that I did not change this, as updated data required a fee and there was a no-scraping notice
+
+replace year = 2022 // correct to proper year
+
+save "mit_living_wage-2022.dta", replace
+
+/*
+*inflate 2022 MIT to 2023 using BLS series CUUR0000SA0. Used first half for 2022, annual for 2021 (keeping this note for prior methods)
+*because both 2022 and 2023 have full annual values, i use the annual here for both
+*because QECW is not available annually for 2023 yet, will use 2022 data so likely won't need this inflation
+
+replace wage = wage* 302.702/292.655 // 2023 CPI-U is first, then 2022 CPI-U	
+
+save "mit_living_wage-2023.dta", replace
+*/
+clear
+
+/***** import QCEW data *****/
+
+cd `raw'
+
+import delimited using "2022_data.csv", numericcols(14 15 16 17 18) varnames(1) clear
+
+/* keep only county totals */
+keep if areatype == "County" & ownership == "Total Covered"
+
+keep st cnty annualaverageweeklywage annualaverageestablishmentcount
+
+rename st state
+
+rename cnty county
+
+destring state, replace // make numeric for merging with MIT data
+
+cd `wages'
+
+/* merge living wage and QCEW data */
+merge 1:m state county using mit_living_wage-2022.dta
+
+*tab county state if _merge == 1*/// two AK counties missing from MIT data
+
+/* drop statewide obs */
+drop if _merge == 1
+
+/* only keep 1 adult, 2 children row */
+keep if adults == "1 Adult" & children == "2 Children"
+
+/* drop duplicates (first two counties repeated) */
+duplicates drop
+
+/* convert living hourly wage to weekly */
+gen weekly_living_wage = wage * 40
+
+/* get ratio (main metric) */
+gen average_to_living_wage_ratio = annualaverageweeklywage/weekly_living_wage
+
+/* create data quality flag 
+per discussion with Greg, >= 30 is 1, <30 is 3 */
+gen wage_ratio_quality = 1 if annualaverageestablishmentcount >= 30 & annualaverageestablishmentcount != .
+replace wage_ratio_quality = 3 if annualaverageestablishmentcount < 30 & annualaverageestablishmentcount != .
+replace wage_ratio_quality = . if annualaverageestablishmentcount == .
+
+/* test flag */
+tab wage_ratio_quality, missing
+
+/* put state and county in string with leading 0s */
+gen new_state = string(state,"%02.0f")
+drop state
+rename new_state state
+
+gen new_county= string(county,"%03.0f")
+drop county
+rename new_county county
+
+/* check ratio */
+/*sum average_to_living_wage_ratio, det
+hist average_to_living_wage_ratio */
+/*tab county state if average_to_living_wage_ratio == 0 */
+
+gen new_ratio = string(average_to_living_wage_ratio)
+drop average_to_living_wage_ratio
+rename new_ratio average_to_living_wage_ratio
```

Comment:
to be consistent with variable conventions in the readme, change to "ratio_average_to_living_wage"

Reply:
Under issue, we renamed this to: ratio_living_wage. So I have updated this variable to have this name (see issue comments).